build/bin/release-next.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+echo "Release Type?"
+read -rp "> " rtype
+
+echo grunt version:${rtype}
+grunt version:${rtype}
+version=$(./build/bin/version)
```

Comment:
I guess we could just have the script cd to the video.js root dir when we start it?